db/base.rb

```diff
@@ -0,0 +1,7 @@
+require 'yaml'
+
+module DB
```

Comment:
I think we should move the db/ folder inside lib/. Everything should be inside lib/ IMO, except for spec/ or any other chore files. This structure I suggest would also let us use always absolute imports, because everything would be defined within the scope of `lib/`

Reply:
ok, moved!